ToDoApp/Controllers/AddItemViewController.swift

```diff
@@ -3,244 +3,351 @@ import UIKit
+      
+    let titleIsEmpty = emptyCheck(titleTextView).isEmpty
+    let memoIsEmpty = emptyCheck(memoTextView).isEmpty
+    
+    if titleIsEmpty || memoIsEmpty { return
+}
+      
+      if let itemToEdit = self.itemToEdit {
+          let editedItem = Info(id: itemToEdit.id,
+                                title: emptyCheck(titleTextView),
+                                priority: PriorityOptions.allCases[priorityChoice.selectedSegmentIndex].rawValue,
+                                memo: emptyCheck(memoTextView),
+                                date: convertDate(),
+                                isCompleted: switchChanged(completedSwitch))
+          mainViewController.editItem(item: editedItem)
+      } else {
+          let newItem = Info(id: mainViewController.tableView.rowsCount + 1,
+                              title: emptyCheck(titleTextView),
+                              priority: PriorityOptions.allCases[priorityChoice.selectedSegmentIndex].rawValue,
+                              memo: emptyCheck(memoTextView),
+                              date: convertDate(),
+                              isCompleted: switchChanged(completedSwitch))
+          mainViewController.addItem(item: newItem)
+      }
+      navigationController?.popViewController(animated: true)
+  }
 
-        navigationController?.popViewController(animated: true)
+  
+  // MARK: - 공백확인 함수
+  func emptyCheck(_ textField: UITextView) -> (String) {
+    guard let info = textField.text, !info.isEmpty else {
+      let alert = UIAlertController(title: "Error",
+                                    message: "Please enter some information",
+                                    preferredStyle: .alert)
+      
+      alert.addAction(UIAlertAction(title: "Dismiss", style: .cancel, handler: nil))
+      present(alert, animated: true, completion: nil)
+      return ""
     }
+    return info
+  }
+  
+
+  // MARK: - 날짜 -> String 변환함수
+  func convertDate() -> String {
+    let dateFormatter = DateFormatter()
+    dateFormatter.dateFormat = "yyyy-MM-dd"
     
-    // MARK: - 공백확인 함수
-    func emptyCheck(_ textField: UITextField) -> (String) {
-        guard let info = textField.text, !info.isEmpty else {
-            let alert = UIAlertController(title: "Error",
-                                          message: "Please enter some information",
-                                          preferredStyle: .alert)
-            
-            alert.addAction(UIAlertAction(title: "Dismiss", style: .cancel, handler: nil))
-            present(alert, animated: true, completion: nil)
-            return "No infomation"
-        }
-        return info
+    let deadLine = dateFormatter.string(from: deadLineDatePicker.date)
+    return deadLine
+  }
+  
+  // MARK: - 스위치 true/false 전달함수
+  func switchChanged(_ sender: UISwitch) -> Bool {
+    if sender.isOn {
+      return true
+    } else {
+      return false
     }
```

Comment:
`isOn` 프로퍼티가 이미 `Bool` 타입이니까 추가적인 로직이 추가되는게 아닌 이상
가독성을 위해 바로 반환해주는게 좋을 것 같아!

```suggestion
    return sender.isOn
```